tools/alpine/Dockerfile

```diff
@@ -25,7 +27,10 @@ RUN cp /mirror/$(uname -m)/APKINDEX.unsigned.tar.gz /mirror/$(uname -m)/APKINDEX
 RUN abuild-sign /mirror/$(uname -m)/APKINDEX.tar.gz
 
 # fetch OVMF for qemu EFI boot (this is not added as a package)
-RUN apk add -X http://dl-cdn.alpinelinux.org/alpine/edge/community ovmf
+RUN mkdir -p /usr/share/ovmf && \ 
```

Comment:
I am trying to generate a per-arch version file, will double check if the ovmf pkg has been recorded in that file like version.x86_64.